static/app/utils/replays/hooks/useCurrentItemScroller.tsx

```diff
@@ -24,13 +24,23 @@ export function useCurrentItemScroller(containerRef: RefObject<HTMLElement>) {
             const element = mutation.target as HTMLElement;
             const isCurrent = element?.ariaCurrent === 'true';
             if (isCurrent && isContainerScrollable() && !isAutoScrollDisabled) {
+              let offset: number;
+
+              // If possible scroll to the middle of the screen instead of to the top
+              if (element.clientHeight < containerEl.clientHeight) {
+                offset =
+                  element.offsetTop -
+                  (containerEl.clientHeight / 2 - element.clientHeight / 2);
+              } else {
+                // Align it to the top as per default if the element is higher than the container
+                offset = element.offsetTop;
+              }
               // Deferring the scroll helps prevent it from not being executed
               // in certain situations. (jumping to a time with the scrubber)
               defer(() => {
-                element?.scrollIntoView({
```

Comment:
Made this change as `scrollIntoView` cannot scroll to multiple elements at the same time, so if the console message and the breadcrumb element where to happen at the same time, one of them wouldn't scroll